app/assets/src/stylesheets/04-templates/templates.scss

```diff
@@ -0,0 +1,22 @@
+header {
+  @include module-colors;
+  margin: 0;
+  padding: 1rem 2rem;
+}
+
+main {
+  @include module-colors;
+  border-radius: 1rem;
+  padding: 1rem;
+  width: 100%;
+  width: calc(100% - 2rem);
+  margin: 1rem auto;
+}
+
+main.--small-centered-content {
+  max-width: 20rem;
+}
+
+main.--primary-layout {
```

Comment:
Would love feedback on this naming scheme. Is meant to be a spin on BEM. Does it work, or is it just weird?

Reply:
What is BEM?

Reply:
Block-Element-Modifier. The `.modal__header--alert` class naming pattern.

Reply:
I mean, knowing nothing about BEM beyond having used it once on a project I like how it works, if you think this innovation has suitable merits to overcome the value of matching consistency across projects then I think we should go with it.

Reply:
I feel like BEM has similar goals to the thing Atomic Design is doing but that the thing Atomic Design is doing accommodates some of the things BEM doesn't do so well a bit better. I'd like to play with less noisy variations on the BEM convention that let us be a bit more tag-centric in our markup, basically.